query-engine/js-connectors/src/proxy.rs

```diff
@@ -591,7 +591,7 @@ mod proxy_test {
         let json_value = serde_json::Value::String(s.to_string());
         let quaint_value = js_value_to_quaint(json_value, column_type);
 
-        let date = NaiveDate::from_ymd(2023, 01, 01);
+        let date = NaiveDate::from_ymd_opt(2023, 01, 01).unwrap();
```

Comment:
Huh, didn't we replace the deprecated `from_ymd` already like a week or two ago? Surprised to see this change again here